# HG changeset patch
# User sbr <sebastien@brissaud.name>
# Date 1375531715 -7200
#      Sat Aug 03 14:08:35 2013 +0200
# Node ID cec3bf5ef20e9aa011278d7ad54e35eda18f114d
# Parent  4989461eedffa9ab64b545a6a6929140484f078d
-Fix (r15790): In CompanyStationsWindow, center the string of the all facilities button

diff --git a/src/station_gui.cpp b/src/station_gui.cpp
--- a/src/station_gui.cpp
+++ b/src/station_gui.cpp
@@ -463,7 +463,7 @@ public:
 
 			case WID_STL_FACILALL: {
 				int cg_ofst = this->IsWidgetLowered(widget) ? 2 : 1;
-				DrawString(r.left + cg_ofst, r.right + cg_ofst, r.top + cg_ofst, STR_ABBREV_ALL, TC_BLACK);
+				DrawString(r.left + cg_ofst, r.right + cg_ofst, r.top + cg_ofst, STR_ABBREV_ALL, TC_BLACK, SA_HOR_CENTER);
 				break;
 			}
 
